prime-router/settings/prod/0163-sc-onboarding-to-up-receiver.yml

```
@@ -0,0 +1,98 @@
+# To submit this to localhost, run:
+#
+# ./prime multiple-settings set --env prod --input ./settings/prod/0163-sc-onboarding-to-up-receiver.yml
+---
+- name: "sc-phd"
+  description: "South Carolina Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "SC"
+  countyName: null
+  filters: []
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "sc-phd"
+      topic: "full-elr"
+      customerStatus: "inactive"
+      translation: !<HL7>
+        schemaName: "metadata/hl7_mapping/receivers/STLTs/SC/SC-receiver-transform"
```

```suggestion
        schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/SC/SC-receiver-transform.yml"
```

I'm in the process of switching these over and don't want bad timing to break this